fastchat/serve/openai_api_server.py

```diff
@@ -269,6 +271,39 @@ async def show_available_models():
     return ModelList(data=model_cards)
 
 
+# TODO: Have check_length and count_tokens share code.
+@app.post("/v1/token_check")
```

Comment:
Is this the OpenAI standard or is this proposed by you? Add a doc string and explain this.